Apps/phthreatgrid/threatgrid.json

```diff
@@ -129,6 +129,11 @@
                         "threatgrid playbook name"
                     ],
                     "primary": true
+                },
+                "file_password": {
+                    "description": "Password used to open the submitted file",
+                    "data_type": "string",
```

Comment:
yes we can

Reply:
on changing the data type to password we got the error `Decryption of parameters[file_password] failed` during action execution, resulting in action failure. Do you have any suggesstion to avoid that?

Reply:
Could you try re-add the asset? Could it be b/c the data_type here mismatch the one below on line [160](https://github.com/phantomcyber/phantom-apps/pull/774/files#diff-f9e6ac54246e1ebb19d1554b1659856032c58bfab22223c73af7fce424776721R160)?

IRRC, with the data_type "password", `param.get('file_password')` should return the variable in string. Using the data_type "password" should simply improve the UI experience in the Asset page of an app I believe.

If that still does not work, please feel free to leave it as `string` and resolve this thread.

cc: @tonyc-phantom

Reply:
as per the suggestion even though after re-adding the asset and matching the data type in line 160 to password, the action is failed with same error "Decryption of parameters[file_password] failed".